notebooks/MAST/TESS/beginner_tess_tap_search/beginner_tess_tap_search.ipynb

```diff
@@ -0,0 +1,375 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connecting to the TAP Service\n",
+    "\n",
+    "The TapPlus library is able to connect to any TAP service, given the \"base\" URL as noted in metadata registry resources describing the service. The CAOM TAP service at MAST has access to TESS FFI and time series, including file URLs for download."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "TAP_service = TapPlus(url=TAP_URL)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use Case: Getting images from a sector, camera, and chip"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 1: Getting the footprint\n",
+    "\n",
+    "For our purposes, any one footprint related to a sector, camera, and chip combination is good enough. We are not currently accounting for small movements of the spacecraft to form a composite footprint. Observation IDs for this mission are constructed based on sector, camera, and chip combination, and we can use this to launch our footprint search:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sector = '1'\n",
+    "camera = '1'\n",
+    "chip = '2'\n",
+    "observationIDwildcard = 'tess%-s000{}-{}-{}-%-s'.format(sector, camera, chip)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we query for a single ID and footprint to start with. For filters, we use the TESS mission, target types associated with light curves instead of full frame images, and the sector number. Note that the sector ID (sequence number) is in a different table in the CAOM database than most of the metadata we want, so we have to join these tables based on the shared observation ID.\n",
+    "\n",
+    "This query can use a synchronous job (with shorter timeouts) because we're only expecting a single line back from a query without time-consuming spatial search calculations or multiple joins of large tables. If you're having connection issues, try changing the launch function to *launch_job_async*."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = TAP_service.launch_job(\"\"\"\n",
+    "            SELECT top 1 obs_id, s_region  \n",
+    "            FROM dbo.caomobservation JOIN ivoa.obscore on dbo.caomobservation.observationID = ivoa.obscore.obs_id \n",
+    "            WHERE collection = 'TESS' and trgType = 'FIELD' and \n",
+    "            sequenceNumber = {} and\n",
+    "            observationID like '{}'\n",
+    "            \"\"\".format(sector, observationIDwildcard))\n",
+    "\n",
+    "#uncomment the line below to see the whole query with wildcards replaced\n",
```

Comment:
Suggest space after '#' for readability. Likewise for other comment lines in notebook.